desks-web-app/src/components/booking-components/WeekOverviewWindow.vue

```diff
@@ -57,7 +95,8 @@ export default defineComponent({
 
     isDisabled: (startOfDay: Moment): boolean => startOfDay.isBefore(moment().startOf("day")),
 
-    calculateNthDayOfWeek(n: number): Moment {
+    calculateNthNextDay(n: number): Moment {
+      console.log(moment(this.startOfWeek).add(n, "day"))
```

Comment:
Please remove this debug log or log a meaning full message.
And don't compute the same value twice